lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -445,7 +445,8 @@ protected function transformSignup($data, $current) {
    */
   protected function transformUser($data) {
     return [
-      'id' => $data->id,
+      'id' => $data->drupal_id,
```

Comment:
@weerd Oh, is `$data->id` already the Northstar ID in this case? Yeah, let's not change that then.